kernel/kernel-impl/src/test/java/org/sakaiproject/content/impl/test/RoleAccessTest.java

```diff
@@ -66,11 +66,17 @@ public void setUp() throws IdUsedException, IdInvalidException, InconsistentExce
         session.setUserId("admin");
 
         _ss = (SiteService)getService(SiteService.class.getName());
-        Site newSite = _ss.addSite(SITE_ID, (String) null);
-        Group group = newSite.addGroup();
-        group.setTitle(".group");
-        _groupReference = group.getReference();
-        _ss.save(newSite);
+        Site newSite = null;
```

Comment:
In my opinion, what's wrong is the way the property is used inside the SiteService methods. For these kind of issues we'd need to set the property to false, but as it's checked directly inside the removeSite method we can't.